### PR TITLE
Initial draft on Shared Framework ownership by Acquisition and Deployment team

### PR DESCRIPTION
**Pull request by rakeshsinghranchi**

### CODE REVIEW

File: Documentation/misc/.NET Core Shared Framework composition - A & D team ownership.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Overview
+
+.NET Acquisition and deployment team will act as arbitrator and work in close liaison with Shared Framework owners. Team will own defining the guidelines to produce and consume a consistent shared framework across .NET Core product. This will greatly help in being proactive to the issues that may come up while composing the product and delivering it to the customers.
+ 
+# Ownership: 
+1. Identify, and report redundancy of  common assemblies carried in multiple Shared Frameworks.
```
Comment (ericstj) [inline Documentation/misc/.NET Core Shared Framework composition - A & D team ownership.md]: nit: whitespace

File: Documentation/misc/.NET Core Shared Framework composition - A & D team ownership.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+2. Drive discussions among Technology owners, team doesn't own the decision or its delivery.
```
Comment (ericstj) [inline Documentation/misc/.NET Core Shared Framework composition - A & D team ownership.md]: what decision or delivery are you talking about?  These terms seem too vague.

Comment (rakeshsinghranchi) [reply Documentation/misc/.NET Core Shared Framework composition - A & D team ownership.md]: Acquisition team owns reporting the assembly redundancy issue, and providing the guidance. Respective technology team ( Shared Framework owner ) owns making the required code change to resolve the issue.

**Comment by dagood** [comment]
Closing stale PR: this doc is out of date now, for example, the infra ownership goals don't match where we are with 3.0. (ASP.NET Core has their own implementation at the moment, and WindowsDesktop's assets are built in the Core-Setup repo.)

When tooling is shared in Arcade, that seems like a better place to describe ownership and scope. https://github.com/dotnet/core-setup/issues/4787

**Pull request closed without merging**